custom-words.txt

```diff
@@ -2724,3 +2724,4 @@ mediumrc
 smallrc
 Autotune
 autogrid
+taginheritance
```

Comment:
Remove this. You won't need it if you use correct casing or the term 'tag inheritance' everywhere.